### ISSUE TITLE
Web - Inconsistent Realtime Updating of Padding and Margin in Tooltip for Long and Short Display Names

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#22112]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Click on the plus sign :heavy_plus_sign:
2. Select "Assign Task" and click "Next"
3. Add a Title and click "Next"
4. Click on Assign and add an email, then confirm tasks
5. Go to settings > profile > Display name and enter a long name
6. Refresh the page
7. Hover over the image associated with the Task Title and observe that the name is written in one line
8. Go to settings again > profile > Display name and enter a short name
9. Hover over the image associated with the Task Title and observe that the padding and margin remain unchanged
10. Write a message in the Task Assign section, hover over the image, and observe that the padding and margin tooltip when hovering over the message is different from the padding and margin associated with images for the Task title.

## Expected Result:
The padding and margin should consistently update in real-time when entering a long or short name, without requiring a page refresh.

## Actual Result:
The padding and margin do not consistently update in real-time when entering a long or short name, requiring a page refresh.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.35-5
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/a33276ca-e2e1-4480-a150-6bd049b590e1

https://github.com/Expensify/App/assets/93399543/2562b975-1f78-4361-bd11-a19b90a1e55c

**Expensify/Expensify Issue URL:**
**Issue reported by:** @tewodrosGirmaA
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1688274688723249

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by sakluger** [comment]
Those do look like the same issue, I'm going to close this one in favor of https://github.com/Expensify/App/issues/21622.

**Comment by tewodrosGirmaA** [comment]
Hi, @sakluger I initially reported this issue on June 15th at https://expensify.slack.com/archives/C049HHMV9SM/p1686843962847709. However, a corresponding GitHub issue was not created for me, and I didn't receive any response. Later on, I reported the same issue again after some time and this time the issue was created for me. I reported it before this report https://github.com/Expensify/App/issues/21622" , Thank you
